doc/dev/adr/0023-large-message-chunking.md

```diff
@@ -0,0 +1,120 @@
+﻿# ADR 23: Large Message Chunking in MQTT Protocol
```

Comment:
Nit - it looks like JD has an ADR 23 out for PR so this will need to be bumped at least by 1.